docs/src/theme/DocSidebar/Desktop/Content/styles.module.css

```
@@ -4,27 +4,26 @@
 
-  .menu[data-sidebar-pane='contextual'] {
-    padding: 0 0.5rem !important;
+  .pane {
+    flex-shrink: 0;
   }
 
-  .pane {
-    transition:
-      transform 150ms cubic-bezier(0.175, 0.885, 0.32, 1.1),
-      opacity 150ms cubic-bezier(0.175, 0.885, 0.32, 1.1);
+  .rootPaneEntry {
```

Stylelint (17.14.0)

[error] Expected class selector ".rootPaneEntry" to be kebab-case (selector-class-pattern)

(selector-class-pattern)